src/conn/conn_api.c

```diff
@@ -2626,14 +2626,35 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler,
 
 	WT_ERR(__wt_config_gets(session, cfg, "file_extend", &cval));
 	for (ft = file_types; ft->name != NULL; ft++) {
+		/*
+		 * An unset log extend length is an indicator to set it
+		 * with configured maximum log file size in case of the default
+		 * configuration. Update the log extend length with actual
+		 * desired size when maximum log file size gets initialized as
+		 * part log server initialization.
+		 */
```

Comment:
I suggest rewording this comment:
* If the log extend length is not set use the default of the configured maximum
* log file size. That size is not known until it is initialized as part of the log
* server initialization.

Reply:
Done.